common/src/main/java/whocraft/tardis_refined/client/renderer/blockentity/device/AstralManipulatorRenderer.java

```diff
@@ -0,0 +1,96 @@
+package whocraft.tardis_refined.client.renderer.blockentity.device;
+
+import com.mojang.blaze3d.vertex.PoseStack;
+import com.mojang.blaze3d.vertex.VertexConsumer;
+import com.mojang.math.Axis;
+import net.minecraft.client.renderer.MultiBufferSource;
+import net.minecraft.client.renderer.RenderType;
+import net.minecraft.client.renderer.blockentity.BlockEntityRenderer;
+import net.minecraft.client.renderer.blockentity.BlockEntityRendererProvider;
+import net.minecraft.client.renderer.texture.OverlayTexture;
+import net.minecraft.core.BlockPos;
+import net.minecraft.resources.ResourceLocation;
+import net.minecraft.world.level.block.state.BlockState;
+import org.joml.Vector3f;
+import whocraft.tardis_refined.client.model.blockentity.console.ConsoleModelCollection;
+import whocraft.tardis_refined.client.model.blockentity.console.ConsoleUnit;
+import whocraft.tardis_refined.client.renderer.RenderHelper;
+import whocraft.tardis_refined.common.block.device.ConsoleConfigurationBlock;
+import whocraft.tardis_refined.common.block.door.GlobalDoorBlock;
+import whocraft.tardis_refined.common.block.shell.ShellBaseBlock;
+import whocraft.tardis_refined.common.blockentity.console.GlobalConsoleBlockEntity;
+import whocraft.tardis_refined.common.blockentity.device.AstralManipulatorBlockEntity;
+import whocraft.tardis_refined.common.blockentity.device.ConsoleConfigurationBlockEntity;
+
+public class AstralManipulatorRenderer implements BlockEntityRenderer<AstralManipulatorBlockEntity>, BlockEntityRendererProvider<AstralManipulatorBlockEntity> {
+
+    public AstralManipulatorRenderer(Context context) {
+    }
```

Comment:
Not a big concern, but it's always good to store the context as a field incase we ever do want to use it
But not worth changing at the moment